bootloader.c

```diff
@@ -130,30 +130,42 @@ bool pv_bootloader_trying_update()
 			 strlen(pv_bootloader_get_rev()) + 1));
 }
 
-int pv_bootloader_set_installed(char *rev)
+int pv_bootloader_install_update(char *rev)
 {
 	pv_log(INFO,
 	       "setting installed revision %s to be started after next reboot",
 	       rev);
+
+	if (ops->install_update)
+		return ops->install_update(rev);
+
 	return pv_bootloader_set_try(rev);
 }
 
-int pv_bootloader_set_commited(char *rev)
+int pv_bootloader_commit_update(char *rev)
 {
 	if (!ops)
 		return -1;
 
 	pv_log(INFO, "setting done revision %s to be started after next reboot",
 	       rev);
+
+	if (ops->commit_update)
+		return ops->commit_update();
+
 	return (pv_bootloader_set_rev(rev) || pv_bootloader_unset_try() ||
 		ops->flush_env());
```

Comment:
in the previous code this was also run for rpiab update... before the commit_update. now we dont run this anymore?